test/appium/tests/medium/test_activity_center.py

```diff
@@ -202,33 +93,56 @@ def prepare_devices(self):
         self.profile_1.just_fyi("Enabling PNs")
         self.profile_1.switch_push_notifications()
 
-    @marks.testrail_id(702850)
-    @marks.xfail(reason="blocked with 15180")
-    def test_activity_center_decline_contact_request_no_pn(self):
+    @marks.testrail_id(702871)
+    def test_activity_center_cancel_outgoing_contact_request_no_pn(self):
+        [home.chats_tab.click() for home in [self.home_1, self.home_2]]
         self.device_1.put_app_to_background()
         self.device_2.just_fyi('Device2 sends a contact request to Device1')
-        self.profile_2.click_system_back_button_until_element_is_shown(self.profile_2.contacts_button)
-        self.profile_2.add_contact_via_contacts_list(self.public_key_1)
-        self.profile_2.click_system_back_button_until_element_is_shown(self.profile_2.contacts_button)
+        self.home_2.add_contact(self.public_key_1)
 
         self.device_1.just_fyi("Device 2: check there is no PN when receiving new message to activity centre")
         self.device_1.open_notification_bar()
         if self.home_1.element_by_text_part("Please add me to your contacts").is_element_displayed():
             self.errors.append("Push notification with text was received for new message in activity centre")
         self.device_1.click_system_back_button(2)
 
-        [home.chats_tab.double_click() for home in [self.home_1, self.home_2]]
+        self.profile_2.just_fyi("Check outgoing contact request and cancel it")
+        for indicator in (self.home_2.notifications_unread_badge, self.home_2.contact_new_badge):
+            if not indicator.is_element_displayed():
+                self.errors.append("Unread indicator on contacts tab or on activity center is not shown for outgoing CR!")
+        self.home_2.contacts_tab.click()
+        if self.home_2.pending_contact_request_text.text != '1':
+            self.errors.append("The amount of contact requests is not shown for outgoing CR!")
+        self.home_2.pending_contact_request_button.click()
+        from views.home_view import ActivityCenterElement
+        outgoing_cr = ActivityCenterElement(self.drivers[1], self.default_username_1)
```

Comment:
Let's move this to the home_view.py creating a method like
`def get_activity_center_element(self, name):
    return ActivityCenterElement(self.driver, name)`
so here it's be like
```suggestion
        from views.home_view import ActivityCenterElement
        outgoing_cr = self.home_1.get_activity_center_element(name=self.default_username_1)
```

Reply:
Ah, this method already exists: `get_chat_from_activity_center_view`

Reply:
fixed